dev-java/protobuf-java/files/protobuf-java-3.22.3-ignore-failing-tests.patch

```diff
@@ -0,0 +1,39 @@
+There were 2 failures:
+1) testWriteWholeMessage(com.google.protobuf.CodedOutputStreamTest)
+java.lang.RuntimeException: Tests data file /google/protobuf/testdata/golden_message_oneof_implemented is missing.
+        at com.google.protobuf.TestUtil.readBytesFromResource(TestUtil.java:3848)
+        at com.google.protobuf.TestUtil.getGoldenMessage(TestUtil.java:3865)
+        at com.google.protobuf.CodedOutputStreamTest.testWriteWholeMessage(CodedOutputStreamTest.java:356)
+2) testWriteWholePackedFieldsMessage(com.google.protobuf.CodedOutputStreamTest)
+java.lang.RuntimeException: Tests data file /google/protobuf/testdata/golden_packed_fields_message is missing.
+        at com.google.protobuf.TestUtil.readBytesFromResource(TestUtil.java:3848)
+        at com.google.protobuf.TestUtil.getGoldenPackedFieldsMessage(TestUtil.java:3881)
+        at com.google.protobuf.CodedOutputStreamTest.testWriteWholePackedFieldsMessage(CodedOutputStreamTest.java:382)
```

Comment:
both these files should be part of a jar so that java can read them as resources. within the jar file the paths should be those as specified in the error. are they really missing from the distributed files or are they just missing in a jar?

Reply:
> both these files should be part of a jar so that java can read them as resources. within the jar file the paths should be those as specified in the error. are they really missing from the distributed files or are they just missing in a jar?

Thanks for the advice.  I only saw these files are expected on an absolute path but didn't think of that path within a JAR.